src/lightbox/tp-lightbox-count.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Internal dependencies.
+ */
+import { TPLightboxElement } from './tp-lightbox';
+
+/**
+ * TP Slider Count.
+ */
+export class TPLightboxCountElement extends HTMLElement {
+	/**
+	 * Get observed attributes.
+	 *
+	 * @return {Array} Observed attributes.
+	 */
+	static get observedAttributes(): string[] {
+		return [ 'format' ];
+	}
+
+	/**
+	 * Get format.
+	 *
+	 * @return {string} Format.
+	 */
+	get format(): string {
+		return this.getAttribute( 'format' ) ?? '$current / $total';
```

Comment:
@junaidbhura This will be 
`return this.getAttribute( 'format' ) || '$current / $total';`

Reply:
Agreed with @thelovekesh the null coalescing operator would be best suited here.

@imranhsayed There's a note about `||` vs `??` here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing#assigning_a_default_value_to_a_variable